### PR TITLE
Auth test 3

### PR DESCRIPTION
**Pull request by Olha98**

### CODE REVIEW

File: backend/package.json [hunk 1]
```diff
@@ -33,15 +33,14 @@
...
+    "faker": "^5.1.0",
     "joi": "^17.3.0",
     "jsonwebtoken": "^8.5.1",
-    "mocha": "^8.2.1",
     "mongoose": "^5.10.14",
     "mongoose-paginate-v2": "^1.3.11",
     "morgan": "^1.10.0",
     "should": "^13.2.3",
     "sinon": "^9.2.1",
-    "supertest": "^6.0.1",
```
Comment (alucardpetrovich) [inline backend/package.json:44]: why you've removed supertest and mocha libs?

File: backend/src/api/auth/auth.controller.js [hunk 2]
```diff
@@ -6,7 +6,8 @@ const jwt = require("jsonwebtoken");
 exports.createNewUser = async (req, res, next) => {
   const { username, email, password } = req.body;
   const existingUser = await UserModel.findOne({ email });
-  console.log(existingUser, "existingUser");
+  // console.log(existingUser, "existingUser!!!!!!!");
```
Comment (alucardpetrovich) [inline backend/src/api/auth/auth.controller.js]: this console.log should be removed completely

File: backend/src/api/errors/error.controller.js [hunk 3]
```diff
@@ -32,6 +32,7 @@ const sendErrorProd = (err, res) => {
 module.exports = (err, req, res, next) => {
   err.statusCode = err.statusCode || 500;
   err.status = err.status || 'error';
+  // console.log("!!500!!", err);
```
Comment (alucardpetrovich) [inline backend/src/api/errors/error.controller.js]: same here

File: backend/src/api/auth/auth.controller.js (hunk 2 above)
Comment (Olha98) [reply backend/src/api/auth/auth.controller.js]: ок

File: backend/src/api/errors/error.controller.js (hunk 3 above)
Comment (Olha98) [reply backend/src/api/errors/error.controller.js]: sorry

File: backend/package.json (hunk 1 above)
Comment (Olha98) [reply backend/package.json:44]: image

Comment (Olha98) [reply backend/package.json:44]: This libs are here.

**Review by alucardpetrovich — APPROVED** [review]

**Pull request merged**